Source/JavaScriptCore/dfg/DFGAbstractInterpreterInlines.h

```diff
@@ -941,6 +942,8 @@ bool AbstractInterpreter<AbstractStateType>::executeEffects(unsigned clobberLimi
     case AtomicsIsLockFree: {
         if (m_graph.child(node, 0).useKind() != Int32Use)
             clobberWorld();
+        else
+            node->clearFlags(NodeMustGenerate);
```

Comment:
We don't need this since it's not even based on the proven type it's not going to do anything.